src/client/Components/Pages/Dashboard.jsx

```diff
@@ -1,18 +1,30 @@
+// @flow
+
 import React from 'react';
 import { Link } from 'react-router-dom';
 
 import validateAuth from '../../util/Util';
 import NavBar from '../Util/NavBar';
 
-export default class Dashboard extends React.Component {
-  constructor(props) {
-    super(props);
+type State = {
```

Comment:
I can add specificity, the only reason I didn't before was because there is only one `State` type per file, as there can only be one class per file. In my mind, it makes it unambiguous as to what it refers to.

Reply:
I think it’s more clear if you specify what it’s the state for. I think it’s a bit of code smell to have a dozen identically named types. It’s also future proofing if the type is ever used elsewhere.